refresh-cache.js

```diff
@@ -0,0 +1,62 @@
+const axios = require("axios");
+const arg_parser = require("args-parser");
+
+// process .env file
+require("dotenv").config();
+const prerender_url = process.env.PRERENDER_URL;
+if (!prerender_url) {
+    console.error("PRERENDER_URL not set in .env file");
+    process.exit(1);
+}
+
+// parse arguments
+const args = arg_parser(process.argv);
+const {
+    ["sitemap-url"]: sitemap_url,
+    ["api-rate-limit"]: api_rate_limit = 500,
+    ["1"]: single_url = undefined,
+} = args;
+
+if (single_url) {
+    refreshCacheForUrl(single_url);
+} else if (sitemap_url) {
+    refreshCacheForSitemap(sitemap_url);
+} else {
+    console.error("No sitemap url or single url provided");
+    process.exit(1);
+}
+
+async function refreshCacheForSitemap(sitemap_url) {
+    const Sitemapper = require("sitemapper");
+    const sitemap = new Sitemapper();
+
+    console.log("fetching sitemap from", sitemap_url);
+    const { sites, errors } = await sitemap.fetch(sitemap_url);
+    console.error("faced errors:", errors);
+
+    console.log(
+        "refreshing cache for",
+        sites.length,
+        "urls from sitemap located at",
+        sitemap_url
+    );
+
+    for (let i = 0; i < sites.length / api_rate_limit; i++) {
+        await Promise.all(
+            sites
+                .slice(i * api_rate_limit, (i + 1) * api_rate_limit)
+                .map(refreshCacheForUrl)
+        );
+        // wait for 1min after api_rate_limit calls
+        await new Promise((resolve) => setTimeout(resolve, 1000 * 60));
```

Comment:
Remove this await as we do not have a rate limit. And set api_rate_limit default to 20.

Reply:
@ravemz I have removed the await but not sure if having the extra promise has any significance if you meant that the whole promise should be removed do tell me